src/node_quirc_decode.c

```diff
@@ -415,3 +421,49 @@ nq_load_jpeg(struct quirc *q, const uint8_t *img, size_t img_len)
 	jpeg_destroy_decompress(&dinfo);
 	return -1;
 }
+
+static int
+nq_load_raw(struct quirc *q, const uint8_t *img, size_t img_len, size_t width, size_t height)
+{
+	if (quirc_resize(q, width, height) < 0)
+		goto fail;
+
+	uint8_t *image = quirc_begin(q, NULL, NULL);
+
+	int channels = img_len / width / height;
+
+	if (channels == 1)
+	{
+		memcpy(image, img, img_len);
+	}
```

Comment:
I don't think the styling was totally consistent in the first place. I think I also got confused because function definitions have `{` on a new line, but `if`/`for`/etc have it on the same line? Anyway, I'll fix it as far as I can. Might be nice to have an auto formatter so this isn't an issue.